sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -139,6 +139,58 @@ describe("LocalWorkspace", () => {
             await workspace.removeStack(stackName);
         });
     });
+    it(`Environment functions`, async () => {
+        // Skipping test because the required environments are in the moolumi org.
+        if (getTestOrg() !== "moolumi") {
+            return;
+        }
+        const projectName = "node_env_test";
+        const projectSettings: ProjectSettings = {
+            name: projectName,
+            runtime: "nodejs",
+        };
+        const ws = await LocalWorkspace.create({ projectSettings });
+        const stackName = fullyQualifiedStackName(getTestOrg(), projectName, `int_test${getTestSuffix()}`);
+        const stack = await Stack.create(stackName, ws);
+
+        let caught = 0;
+
+        // Adding non-existent env should fail.
+        try {
+            await stack.addEnvironments("non-existent-env");
+        } catch (error) {
+            caught++;
+        }
+        assert.strictEqual(caught, 1, "expected adding a non-existent env to throw");
```

Comment:
This and others below could be simplified as:

```suggestion
        await assert.rejects(stack.addEnvironments("non-existent-env"));
```